payment_ippay/models/payment.py

```diff
@@ -9,15 +10,22 @@
 
 
 class PaymentAcquirerIppay(models.Model):
+    """Ippay Payment acquirer."""
+
     _inherit = "payment.acquirer"
 
     provider = fields.Selection(selection_add=[("ippay", "IPpay")])
-    api_url = fields.Char("Api URL", required_if_provider="ippay")
-    ippay_terminal_id = fields.Char("IPpay TerminalID", required_if_provider="ippay")
+    api_url = fields.Char(
+        "Api URL",
+        required_if_provider="ippay")
+    ippay_terminal_id = fields.Char(
+        "IPpay TerminalID",
+        required_if_provider="ippay")
 
     @api.model
     def ippay_s2s_form_process(self, data):
-        Token = self.env["payment.token"]
+        """Get Payment token ref from Ippay payment token."""
+        token = self.env["payment.token"]
```

Comment:
@hpatelserpentcs The convention is for object references to use Title uppercase names, like "Token". Lowercase "token" should be used for object instances (recordsets).

Reply:
But it throws the flack8: variable in function should be lowercase